Projects/UOContent/Mobiles/BaseCreature.cs

```diff
@@ -3062,7 +3066,7 @@ public static List<DamageStore> GetLootingRights(List<DamageEntry> damageEntries
             if (rights.Count > 0)
             {
                 // This would be the first valid person attacking it.  Gets a 25% bonus.  Per 1/19/07 Five on Friday
-                rights[0].m_Damage = (int)(rights[0].m_Damage * 1.25);
+                rights[rights.Count - 1].m_Damage *= (int) 1.25;
```

Comment:
Apparently this is a really old bug from RunUO circa 2009! Nice catch!

Reply:
This is actually _not correct_, since expired entries _are removed_. So we need a more "proper" way to calculate this.
Also _when was this introduced_?